src/message.ts

```diff
@@ -1,13 +1,20 @@
 import {CommitHash, nullCommitHash} from './commit-hash'
 import {MessageKey} from './message-key'
+import {NO_JOB_ID} from './job'
 
 export abstract class Message {
   payload: string
   jobRef: CommitHash
+  id: number
```

Comment:
@yeraydavidrodriguez I would rename it from `id` to `jobId` because you might think it's the message ID. And change the type from `number` to `JobId` when you add that new type.